content/articles/2019-09-devtools-2.2.0.Rmarkdown

```diff
@@ -0,0 +1,96 @@
+---
+title: devtools 2.2.0
+date: 2019-09-06
+slug: devtools-2-2-0
+author: Jim Hester
+categories: [package]
+description: >
+    Make package development easier by providing R functions that simplify and expedite common tasks.
+photo:
+  url: https://unsplash.com/photos/wEJK4q_YlNQ
+  author: Hunter Haley
+---
+
+## Introduction
+
+[devtools] 2.2.0 is now on CRAN!
+
+devtools makes package development easier by providing R functions that
+simplify and expedite common tasks. [R Packages] is a book based around this
+workflow.
```

Comment:
Can you do more along these lines:

R Packages is a book based around this workflow, which you can read online for free ([1st edition](http://r-pkgs.had.co.nz/), in-progress [2nd edition](https://r-pkgs.org)).